### PR TITLE
Add debianization for buster [REVPI-3198]

### PR DESCRIPTION
**Pull request by tboehler1**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -0,0 +1,64 @@
+pitest (1.4.0-1+revpi10+1) buster; urgency=medium
+
+  * Add MIT license in LICENSE.txt
+  * Add piControl dependency, correct Makefile
+  * Change build system to CMake
+  * Add README.md with help output of binary
+  * fix: Add missing man page
+  * Initial debianization
+  * Assign correct license to sources
```
Comment (iluminat23) [inline debian/changelog:9]: Also relevant changes to piTest done in the old Repo are missing:
```
  * piTest: Add Connect 4 as known device
  * Add SPDX headers
  * Replace strncpy by snprintf
  * Fix gcc -Wformat= complaints
```

The are the changes done to the `piTest/` directory sind the last release (see last change to `debian/changelog`).

Comment (tboehler1) [reply debian/changelog:9]: I added the commit mentioned above as changes to the `debian/` directory were previously also included in the changelog.

File: debian/rules [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/make -f
+
+# see FEATURE AREAS in dpkg-buildflags(1)
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
+# see ENVIRONMENT in dpkg-buildflags(1)
+# package maintainers to append CFLAGS
+export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
+# package maintainers to append LDFLAGS
+export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
```
Comment (naruxde) [inline debian/rules:10]:
```suggestion
export DEB_BUILD_MAINT_OPTIONS = hardening=+all
export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
```

**Review by naruxde — CHANGES_REQUESTED** [review]
Can not build the raspios/buster branch.
```
   dh_auto_install
	cd obj-arm-linux-gnueabihf && make -j6 install DESTDIR=/work/revpi-pitest/debian/pitest AM_UPDATE_INFO_DIR=no "INSTALL=install --strip-program=true"
make[1]: Entering directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
/usr/bin/cmake -S/work/revpi-pitest -B/work/revpi-pitest/obj-arm-linux-gnueabihf --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /work/revpi-pitest/obj-arm-linux-gnueabihf/CMakeFiles /work/revpi-pitest/obj-arm-linux-gnueabihf/CMakeFiles/progress.marks
make -f CMakeFiles/Makefile2 all
make[2]: Entering directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
make -f src/CMakeFiles/piControlReset.dir/build.make src/CMakeFiles/piControlReset.dir/depend
make -f src/CMakeFiles/piTest.dir/build.make src/CMakeFiles/piTest.dir/depend
make[3]: Entering directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
cd /work/revpi-pitest/obj-arm-linux-gnueabihf && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /work/revpi-pitest /work/revpi-pitest/src /work/revpi-pitest/obj-arm-linux-gnueabihf /work/revpi-pitest/obj-arm-linux-gnueabihf/src /work/revpi-pitest/obj-arm-linux-gnueabihf/src/CMakeFiles/piControlReset.dir/DependInfo.cmake --color=
make[3]: Entering directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
cd /work/revpi-pitest/obj-arm-linux-gnueabihf && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /work/revpi-pitest /work/revpi-pitest/src /work/revpi-pitest/obj-arm-linux-gnueabihf /work/revpi-pitest/obj-arm-linux-gnueabihf/src /work/revpi-pitest/obj-arm-linux-gnueabihf/src/CMakeFiles/piTest.dir/DependInfo.cmake --color=
make[3]: Leaving directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
make[3]: Leaving directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
make -f src/CMakeFiles/piControlReset.dir/build.make src/CMakeFiles/piControlReset.dir/build
make -f src/CMakeFiles/piTest.dir/build.make src/CMakeFiles/piTest.dir/build
make[3]: Entering directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
cd /work/revpi-pitest/obj-arm-linux-gnueabihf/src && ln -s piTest /work/revpi-pitest/obj-arm-linux-gnueabihf/piControlReset
make[3]: Entering directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
make[3]: Nothing to be done for 'src/CMakeFiles/piTest.dir/build'.
make[3]: Leaving directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
ln: failed to create symbolic link '/work/revpi-pitest/obj-arm-linux-gnueabihf/piControlReset': File exists
make[3]: *** [src/CMakeFiles/piControlReset.dir/build.make:60: src/CMakeFiles/piControlReset] Error 1
make[3]: Leaving directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
make[2]: *** [CMakeFiles/Makefile2:96: src/CMakeFiles/piControlReset.dir/all] Error 2
make[2]: *** Waiting for unfinished jobs....
[100%] Built target piTest
make[2]: Leaving directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
make[1]: *** [Makefile:133: all] Error 2
make[1]: Leaving directory '/work/revpi-pitest/obj-arm-linux-gnueabihf'
dh_auto_install: cd obj-arm-linux-gnueabihf && make -j6 install DESTDIR=/work/revpi-pitest/debian/pitest AM_UPDATE_INFO_DIR=no "INSTALL=install --strip-program=true" returned exit code 2
make: *** [debian/rules:13: binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
debuild: fatal error at line 1182:
```

**Comment by tboehler1** [comment]
The build is fixed with (and thus blocked on) PR #5

**Review by naruxde — APPROVED** [review]
Package build: OK
Update on target system: OK
```
Preparing to unpack .../6-pitest_1.4.0-1+revpi10+1_armhf.deb ...
Unpacking pitest (1.4.0-1+revpi10+1) over (1.3.1-1) ...
Setting up pitest (1.4.0-1+revpi10+1) ...
```

**Comment by tboehler1** [comment]
This is blocked by PR #7.

**Review by iluminat23 — APPROVED** [review]

**Pull request merged**